docs/chain/delegator.md

```diff
@@ -30,28 +30,72 @@ A Delegator can increase and decrease the stake by calling either  `delegator_st
 
-1. Using the selected account: Select the Delegator Account.
-2. Submit the following extrinsic: parachainStaking -> delegator_stake_more or delegator_stake_less.
-3. Selecting the collator account: Choose the desired stake amount.
+1. Use the selected account: Select the Delegator Account.
+2. Submit the following extrinsic: parachainStaking -> `delegator_stake_more` or `delegator_stake_less`.
+3. Select the collator account: Choose the desired stake amount.
 
-## Revoking
+## How to revoke
 
-A delegator revokes the delegation by calling `revoke_delegation`, reducing the delegation by the full amount of the delegation.
+A delegator revokes their delegation by calling `revoke_delegation`, reducing the delegation by the full amount of the delegation.
 
 ![](https://i.imgur.com/nf5NgLs.png)
 
-- The amount will be prepared for unstaking.
+- The corresponding amount will be prepared for unstaking.
 - Revoking a delegation does not count towards the limit of “1 delegation per round”.
 
-## How to Exit
+## How to exit
 
 An account can revoke all delegations at once by calling `leave_delegators`.
 
 ![](https://i.imgur.com/xemN4fu.png)
 
-Once the canditate exits, by delegation is removed and the amount is prepared for unstaking by calling `execute_leave_candidates`.
+Once a collator candidate exits by calling `execute_leave_candidates`, all of their delegations are automatically removed and the corresponding amounts are prepared for unstaking.
 
-![](https://i.imgur.com/holdTKq.png)
+## How to unlock unstaked tokens
 
+![](https://i.imgur.com/holdTKq.png)
 
-A revoking delegation has to wait 7 days before collecting the staked amount back.
+A revoking delegation has to wait 7 days before unlocking the staked amount back by executing `unlock_unstaked`.
+
+## Lifecycle of a Delegator
+
+The following diagram depicts the lifecycle of a delegator from owning free KILT to delegating, losing a delegation seat, re-delegating and finally unlocking their stake.
+
+```mermaid
+flowchart TD
+   A["Hold at least 20 KILT"] --> |chose candidate| B("Collator Candidate chosen")
+   B --> |join_delegators| C{"Can delegate to target? \n Either \n 1. There are empty \n delegations or \n 2. You delegate more \n than another delegator"}
+   C --> |yes| D("Delegating to a Collator Candidate")
+   C --> |no| C2{"Balance locked?\n E.g. previously delegated \n without unlocking?"}
+   C2 --> |no| A
+   C2 --> |yes| G 
+   D --> |"leave \n delegators"| E("Not delegating")
+   D --> |"revoke \n delegation"| E
+   D --> |"your collator \n candidate leaves"| E
+   E --> F{"Delegate to \n another candidate?"}
+   F --> |yes| B
+   F --> |no| G("Locked tokens")
+   G --> |want to unlock| H{"Waited 7 days?"}
+   H --> |yes| I("Balance with expired lock")
+   H --> |no| F
+   I --> |unlock_unstaked| A
+
+       %% Styles
+    A:::spearmint
+    B:::yellow
+    C:::yellow
+    C2:::rose
+    D:::green
+    E:::rose
+    F:::rose
+    G:::red
+    H:::red
+    I:::red
```

Comment:
Ah yes the colors should than probably have the same meaning as in the collator guide. A table that explains the colors would be helpful.

Reply:
Maybe even call the styles not like the color, but what their meaning is! E.g. not `red` but `lockedCoinsNoRewards`

Reply:
[72d0101](https://github.com/KILTprotocol/docs/pull/64/commits/72d010156e160e3e21c394e1b7b9eaf2611b883a)